WebRoot/Templates/navbar admin.php

```diff
@@ -5,12 +5,14 @@
 		//'/admin/users' => ['Users', 'users.svg'],
 		'/admin/tags' => ['Tags', 'tag.svg'],
 		'/admin/database' => ['Database', 'dns.svg'],
-		'/admin/settings' => ['Settings', 'label.svg']
+		'/admin/settings' => ['Settings', 'slider.svg']
 	];
 ?>
 <div class="sticky">
 	<div class="navbar-box" id="navbar">
-		<img class="panel-ribbon" src="/static/logo/ribbon.svg" alt="wuwana logo ribbon">
+		<div class="panel-ribbon">
+			<img src="/static/logo/ribbon.svg" alt="wuwana logo ribbon">
```

Comment:
This image should have alt="" to indicates that is not a key part of the content